ios/RN/RNCamera.m

```diff
@@ -864,9 +866,11 @@ - (void)takePicture:(NSDictionary *)options resolve:(RCTPromiseResolveBlock)reso
         
         @try {
             AVCapturePhotoSettings *photoSettings = [AVCapturePhotoSettings photoSettingsWithFormat:@{AVVideoCodecKey : AVVideoCodecTypeHEVC}];
+            [photoSettings setFlashMode:self.flashMode];
```

Comment:
nice, is flash mode supported only for iOS11? should we also check if isFlashModeSupported: https://developer.apple.com/documentation/avfoundation/avcapturedevice/1386434-isflashmodesupported